opencti-platform/opencti-graphql/src/domain/stixRefRelationship.js

```diff
@@ -48,14 +57,36 @@ export const isDatable = (entityType, relationshipType) => {
 };
 
 // Mutation
-
+// @Deprecated method.
+// updateField must be directly used
```

Comment:
I did not find this method in the project. What are you referring to ?

Is it a TODO ?